### PR TITLE
Move files into src folder on backend

### PR DESCRIPTION
**Pull request by dawid-pstrak-rmtcfm-com**

### CODE REVIEW

File: backend/src/public/swagger/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```
Comment (ewelina-szczotka-prpl) [inline backend/src/public/swagger/index.html]: it should be in gitingore

Comment (dawid-pstrak-rmtcfm-com) [reply backend/src/public/swagger/index.html]: done

**Review by rafal-ksiazek-rmtcfm-com — APPROVED** [review]

File: backend/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const path = require('path');
```
Comment (rafal-ksiazek-rmtcfm-com) [inline backend/index.js]: this whole file also should be moved to src/

File: backend/db-setup.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ const {
         port,
         auth: { user, pass }
     }
-} = require('./config');
+} = require('/src/config');
```
Comment (rafal-ksiazek-rmtcfm-com) [inline backend/db-setup.js]: also this whole file should be moved to /src/scripts

and we should run this script by "npm run dbsetup" so we should update package.json

File: backend/index.js (hunk 2 above)
Comment (dawid-pstrak-rmtcfm-com) [reply backend/index.js]: done

File: backend/db-setup.js (hunk 3 above)
Comment (dawid-pstrak-rmtcfm-com) [reply backend/db-setup.js]: done

**Pull request merged**